src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,38 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage{
+
+    public LoginPage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+   By emailField = By.cssSelector("input[type='email']");
+    By passwordField = By.cssSelector("input[type='password']");
+    By submitBtn= By.cssSelector("input[type='email']");
+
+    public void provideEmail (String email ) {
+        findElement(emailField).sendKeys(email);
+
+    }
+
+    public void providePassword(String password ){
+        findElement(passwordField).sendKeys(password);
+    }
+
+    public void clickSubmit(){
+        findElement(submitBtn).click();
+    }
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._